### ISSUE TITLE
open downloads tab with :downloads

### ISSUE THREAD

**Issue by jcrowgey** [cmcaine/tridactyl#615]
vimperator opened the downloads (I think in a new tab) when you ran `:downloads`.  That functionality would be nice to have.

**Comment by glacambre** [comment]
What Pentadactyl does is fill the command line with info about your downloads, which is blocked by the rewrite of the UI for Tridactyl. Vimperator and Pentadactyl might have behaved differently but I'd be surprised if they did.

If all you need is opening the downloads in a new tab, you could `:alias downloads tabopen about:downloads`

**Comment by Flimm** [comment]
Running `:tabopen about:downloads` doesn't work for me, and neither does the alias. I'm on Firefox 90.0.

**Comment by bovine3dom** [comment]
You need to have `:native` installed and working : )
